### PR TITLE
Update instructions for running the project from the scratch

### PR DESCRIPTION
**Pull request by piotrpo**
README updated basing on my experiences while running the project from scratch.

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @piotrpo, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;android#837).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: README.md [hunk 1]
```diff
@@ -11,15 +11,28 @@ If you are looking for documentation around the companion applications check out
 
 - Create a Firebase project at [Firebase Console](https://console.firebase.google.com)
 
-- Create two Android apps, one with `io.homeassistant.companion.android` and one with `io.homeassistant.companion.android.debug` as package name
+- Create four Android apps, with following package names 
+ - `io.homeassistant.companion.android`
+ - `io.homeassistant.companion.android.debug`
+ - `io.homeassistant.companion.android.minimal`
+ - `io.homeassistant.companion.android.minimal.debug`
```
Comment (JBassett) [inline README.md:18]: You should mock out the minimal packages and not need to add them in firebase.

File: README.md [hunk 2]
```diff
@@ -11,15 +11,28 @@ If you are looking for documentation around the companion applications check out
...
 
-- Now download the `google-services.json` file and put it in the _home-assistant-Android/app_ folder
+- Now download the `google-services.json` file and put it in the _home-assistant-Android/app_ folder. This file contains the configuration of the whole project (all the four applications).
 
   [You can also use the mock services file instead of generating your own](/.github/mock-google-services.json)
-  The file should contain client IDs for `io.homeassistant.companion.android` and `io.homeassistant.companion.android.debug` for debugging to work properly.  **If you do not generate your own file push notification will never work**
+  The file should contain client IDs for all packages listed above for debugging to work properly.  **If you do not generate your own file push notification will never work**
```
Comment (JBassett) [inline README.md:23]: If you just mock the minimal out you should be fine.

File: README.md [hunk 3]
```diff
@@ -11,15 +11,28 @@ If you are looking for documentation around the companion applications check out
...
 - Start Android Studio, open your source code folder and check if the Gradle build will be successful
 
+- Create keystore containing keypair for debug application signing. In Android Studio: Menu/Build/Generate signed APK, then use a button to create new keystore. Remember the passwords and key alias
+
+- Set environmental variables used in `app/build.gradle.kts`:
+ - `KEYSTORE_PASSWORD`
+ - `KEYSTORE_ALIAS`
+ - `KEYSTORE_ALIAS_PASSWORD`
+ 
+- Run the build using `gradlew buld` from command line.
```
Comment (JBassett) [inline README.md:34]: This assumes Linux, Windows has a different command, might want them just to use the IDE.

```suggestion
- Run the build using `gradlew build` from command line.
```

File: README.md [hunk 4]
```diff
@@ -11,15 +11,28 @@ If you are looking for documentation around the companion applications check out
...
+  The file should contain client IDs for all packages listed above for debugging to work properly.  **If you do not generate your own file push notification will never work**
 
 - Start Android Studio, open your source code folder and check if the Gradle build will be successful
 
+- Create keystore containing keypair for debug application signing. In Android Studio: Menu/Build/Generate signed APK, then use a button to create new keystore. Remember the passwords and key alias
+
+- Set environmental variables used in `app/build.gradle.kts`:
+ - `KEYSTORE_PASSWORD`
+ - `KEYSTORE_ALIAS`
+ - `KEYSTORE_ALIAS_PASSWORD`
```
Comment (JBassett) [inline README.md:32]: Did you have to create this?  Isn't one auto generated for you?

File: README.md (hunk 1 above)
Comment (piotrpo) [reply README.md:18]: When I tried to build it, I got an compile error with message about these non-minimal packages missing.

File: README.md (hunk 2 above)
Comment (piotrpo) [reply README.md:23]: As above - you'll be probably able to build some flavors, but full build won't be possible.

File: README.md (hunk 4 above)
Comment (piotrpo) [reply README.md:32]: Variables won't be created / set automatically. The configuration uses empty strings if env. vars not found. I had to create the debug keystore manually - maybe just some problem with my IDE / project import.

**Pull request merged**